src/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import UserProfile from './components/UserProfile.jsx';
+import NotFoundPage from './containers/NotFoundPage';
+
+class App extends Component {
+  render() {
+    return (
+      <>
```

Comment:
@TheDrizzyWay I guess you used this because you needed a wrapper to for all the components within.
 I think you can consider wrapping them inside `<React.Fragment></React.Fragment>`. I think it serves as a perfect dummy wrapper.

Reply:
This is actually a shorthand for react fragment.
https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

Reply:
Alright